src/Http/Http.Results/src/Results.cs

```diff
@@ -0,0 +1,1243 @@
+        /// <returns>The created <see cref="CreatedAtRouteResult"/> for the response.</returns>
+        public static IResult CreatedAtRoute(object? routeValues, object? value)
+            => CreatedAtRoute(routeName: null, routeValues: routeValues, value: value);
+
+        /// <summary>
+        /// Creates a <see cref="CreatedAtRouteResult"/> object that produces a <see cref="StatusCodes.Status201Created"/> response.
+        /// </summary>
+        /// <param name="routeName">The name of the route to use for generating the URL.</param>
+        /// <param name="routeValues">The route data to use for generating the URL.</param>
+        /// <param name="value">The content value to format in the entity body.</param>
+        /// <returns>The created <see cref="CreatedAtRouteResult"/> for the response.</returns>
+        public static IResult CreatedAtRoute(string? routeName, object? routeValues, object? value)
+            => new CreatedAtRouteResult(routeName, routeValues, value);
+
+        #endregion
+
+        #region AcceptedResult
+        /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        public static IResult Accepted()
+            => new AcceptedResult();
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <param name="value">The optional content value to format in the entity body; may be null.</param>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        public static IResult Accepted(object? value)
+            => new AcceptedResult(location: null, value: value);
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <param name="uri">The optional URI with the location at which the status of requested content can be monitored.
+        /// May be null.</param>
+        /// <returns>The created <see cref="AcceptedResult"/> for the response.</returns>
+        public static IResult Accepted(Uri uri)
+        {
+            if (uri == null)
+            {
+                throw new ArgumentNullException(nameof(uri));
+            }
+
+            return new AcceptedResult(locationUri: uri, value: null);
+        }
+
+        /// <summary>
+        /// Creates a <see cref="AcceptedResult"/> object that produces an <see cref="StatusCodes.Status202Accepted"/> response.
+        /// </summary>
+        /// <param name="uri">The optional URI with the location at which the status of requested content can be monitored.
+        /// May be null.</param>
```

Comment:
We're not consistent with how we document nullable ref types in params. Some are 'blah blah; may be null', others are like this one. Let's just pick one form and stick with it.